### PR TITLE
Added RazorLight to Template Engines

### PR DESCRIPTION
**Pull request by DenisPav**
Added https://github.com/toddams/RazorLight

### CODE REVIEW

**Comment by ngohungphuc** [comment]
@DenisPav please resolve your conflict then I will merge your pr

File: README.md [hunk 1]
```diff
@@ -497,6 +497,7 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [dotliquid](https://github.com/dotliquid/dotliquid) - .NET Port of Tobias Lütke's Liquid template language.
 * [fluid](https://github.com/sebastienros/fluid) - Open-source .NET template engine that is as close as possible to the Liquid template language.
 * [Razor](https://github.com/aspnet/Razor) - Parser and code generator for CSHTML files used in view pages for MVC web apps.
+* [RazorLight](https://github.com/toddams/RazorLight) - Template engine based on Microsoft's Razor parsing engine for .NET Core.
```
Comment (ngohungphuc) [inline README.md]: @DenisPav please resolve your conflict then I will merge your pr

**Comment by DenisPav** [comment]
@ngohungphuc, it should be fine now

**Comment by thangchung** [comment]
@DenisPav Thank you Denis 👍

**Pull request merged**